package.json

```diff
@@ -310,14 +318,7 @@
       "typings"
     ],
     "nohoist": [
-      "@fluentui/react-northstar-fela-renderer/stylis",
-      "@fluentui/react-northstar-emotion-renderer/@types/stylis",
-      "@fluentui/react-northstar-emotion-renderer/stylis",
-      "@fluentui/web-components/@microsoft/eslint-config-fast-dna",
-      "@fluentui/web-components/@storybook/html",
-      "@fluentui/web-components/ts-loader",
-      "@fluentui/web-components/ts-loader/**",
-      "@fluentui/web-components/webpack"
+      "@fluentui/web-components/@storybook/html"
```

Comment:
this is needed until we gonna use nx storybook executor or custom alias to determine framework (html/react/angular) etc